web/src/pages/User/components/LoginMethodPassword.tsx

```diff
@@ -102,10 +102,9 @@ const LoginMethodPassword: UserModule.LoginMethod = {
   submit: async ({ username, password }) => {
     if (username !== '' && password !== '') {
       try {
-        const result = await request('/apisix/admin/user/login', {
+        const result = await request('/user/login', {
```

Comment:
Do we need to update related test cases?

Reply:
request will handle this :)